src/popup.cpp

```diff
@@ -120,7 +123,7 @@ void query_popup_impl::on_resized()
                 for( const auto &opt : line ) {
                     button_width += get_text_width( remove_color_tags( opt ) );
                 }
-                // Right align.
+                // Righzt align.
```

Comment:
```suggestion
                // Right align.
```

Reply:
this is what i get for coding with a bird walking all over my keyboard